features/comments_and_kudos/inbox.feature

```diff
@@ -159,3 +159,8 @@ Feature: Get messages in the inbox
       And I go to the homepage
     Then I should see "sewwiththeflo on Cat Thor's Bizarre Adventure"
       And I should see "Thank you! Please go to bed."
+
+   Scenario: Inbox has the expected browser page title
+    Given I am logged in as "boxer"
+    When I go to boxer's inbox page
+    Then I should see the page title "Show Inbox | Example Archive"
```

Comment:
I'd remove the "Example Archive" because it will cause the tests to fail if you change it locally (which I've done).

Reply:
I'd really like to check the entire title, not just the part before the pipe, so I'll update the tests to set the app name. Do you get the failure in [this challenge test](https://github.com/otwcode/otwarchive/blob/4418571dfa0dc5e00438a005937ae1e7dc365e9b/features/gift_exchanges/challenge_yuletide.feature#L591), too? I don't think I've ever run into it even though my local.yml sets `APP_NAME: Development Archive`, but it's possible I've never run that test locally.

Reply:
Yes, the challenge test failed for me before, that's why I knew it was a potential issue.

Nice solution!